felles/abac-kontekst/src/main/java/no/nav/vedtak/sikkerhet/populasjon/TilgangKlient.java

```diff
@@ -54,44 +54,41 @@ public Set<AnsattGruppe> vurderAnsattGrupper(UUID ansattOid, Set<AnsattGruppe> p
     }
 
     @Override
-    public Tilgangsvurdering vurderTilgangInternBruker(UUID ansattOid, Set<String> personIdenter, Set<String> aktørIdenter, String saksnummer) {
-        var request = new PopulasjonInternRequest(ansattOid, personIdenter, aktørIdenter, saksnummer);
+    public Tilgangsvurdering vurderTilgangInternBruker(UUID ansattOid, Set<String> identer, String saksnummer, UUID behandling) {
+        var request = new PopulasjonInternRequest(ansattOid, identer, saksnummer, behandling);
         var rrequest = RestRequest.newPOSTJson(request, internBrukerUri, restConfig).timeout(Duration.ofSeconds(3));
-        var resultat = klient.send(rrequest, TilgangsvurderingDto.class);
-        return resultat.tilTilgangsvurdering();
+        var resultat = klient.send(rrequest, TilgangsvurderingRespons.class);
+        return mapTilgangsvurdering(resultat);
     }
 
     @Override
-    public Tilgangsvurdering vurderTilgangEksternBruker(String subjectPersonIdent, Set<String> personIdenter, Set<String> aktørIdenter, int aldersgrense) {
-        var request = new PopulasjonEksternRequest(subjectPersonIdent, personIdenter, aktørIdenter, aldersgrense);
+    public Tilgangsvurdering vurderTilgangEksternBruker(String subjectPersonIdent, Set<String> identer, int aldersgrense) {
+        var request = new PopulasjonEksternRequest(subjectPersonIdent, identer, aldersgrense);
         var rrequest = RestRequest.newPOSTJson(request, eksternBrukerUri, restConfig).timeout(Duration.ofSeconds(3));
-        var resultat = klient.send(rrequest, TilgangsvurderingDto.class);
-        return resultat.tilTilgangsvurdering();
+        var resultat = klient.send(rrequest, TilgangsvurderingRespons.class);
+        return mapTilgangsvurdering(resultat);
     }
 
     private record GruppeDto(@NotNull @Valid Set<AnsattGruppe> grupper) { }
 
 
     private record UidGruppeDto(@NotNull UUID uid, @Valid @NotNull Set<AnsattGruppe> grupper) { }
 
-    private record PopulasjonInternRequest(UUID ansattOid,
-                                          Set<String> personIdenter,
-                                          Set<String> aktørIdenter,
-                                           String saksnummer) {
-    }
+    public record PopulasjonInternRequest(@NotNull UUID ansattOid,
```

Comment:
Her gjøres det vel ikke noe hibernate validering og disse annoteringene vil derfor ikke ha noe effekt. Gjelder alle requestene her (?)

Reply:
Eller gjøres det noe validering lenger inn?

Reply:
Det trengs egentlig ikke validering her - mer for veiledning.
Kan legge på noen flere NotNull.
FP-tilgang trenger mer av valideringene og er ganske defensiv.
Ligger Dtos her og i FP-tilgang. 
De kommer til å være stabile framover, så lite poeng i en kontraktsmodul

Reply:
Tenkte meg det var til veiledningsformål. Tenker det er greit 🆗